scripts/affected-libs.sh

```diff
@@ -58,7 +54,7 @@ if [ ! -f $DIRECTORY/deps.txt ]; then
     npm run affected:libs -- $HEAD_SHA_BRANCH "HEAD" > $DIRECTORY/deps.txt
 fi
 
-cat $DIRECTORY/deps.txt
+#cat $DIRECTORY/deps.txt
```

Comment:
do we need that or can be removed?

Reply:
it can be removed it was used on debugging time to make sure the file content was the expected one